Sources/Web3Core/EthereumNetwork/Utility/HexDecodable+Extensions.swift

```diff
@@ -8,15 +8,15 @@
 import Foundation
 import BigInt
 
-extension Int: LiteralInitiableFromString { }
+extension Int: LiteralInitableFromString { }
```

Comment:
Please undo this changes, as they affecting code, rather then comments and should be reviewed separately.